debian/copyright

```diff
@@ -0,0 +1,47 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: citus
+Source: https://github.com/citusdata/citus
+
+License: AGPL-3.0+
```

Comment:
I do not think that AGPL license apply for azure_gdpr. Currently rpm specs also appear to contain wrong license fields. However I am not 100% sure, and it may be better to get some legal advice here.

I think we should drop the copyright file from here, and add an appropriate one later.

This also raises the question: Why do we add a copyright notice only in the debian package, and leave out rpm packages?

Reply:
I changed copyright into commercial in accordance with the comments in Citus Engine chat which suggest using Citus Enterprise License